### ISSUE TITLE
Framework versions

### ISSUE THREAD

**Issue by isaacbernat** [somebee/todomvc-render-benchmark#1]
**TL;DR** It would be handy to display which versions of frameworks are used in the benchmarks.
## 

In the readme you acknowledge _There has been a TodoMVC benchmark floating around earlier_. [Those benchmarks](http://lhorie.github.io/todomvc-perf-comparison/todomvc-benchmark/) had the framework version printed next to the test, so one can see for example that the React test there uses 0.10. [The next version 0.11 was released in July 2014](https://facebook.github.io/react/blog/2014/07/17/react-v0.11.html) so one can estimate how outdated it is. I see you use 0.13 for React, which is rather recent, as [React 0.14 was released in October 2015](https://facebook.github.io/react/blog/2015/10/07/react-v0.14.html). 

IMHO, seeing all versions are recent makes the test appear more relevant and it helps create trust that the comparison with other frameworks is fair. One could argue that anybody could see versions by digging into the code in the repository, but this may be too much friction for most.

**Comment by somebee** [comment]
Very good point, I will add that. I should also update React to use 0.14, I think it might perform better as well.
